tests/test_auth.py

```diff
@@ -254,3 +259,76 @@ def test_config_auth_type_from_env(monkeypatch):
 
     assert cfg.auth_type == 'basic'
     assert cfg.host == 'https://x'
+
+
+def get_test_server(host: str, token: str, expires_after: int):
```

Comment:
```
# These are auto-generated tests for Unified Authentication
# In case of editing this file, make sure the change is propagated to all Databricks SDK codebases
```